packages/ui/src/working-groups/components/WorkingGroupImagePlaceholder.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+import styled from 'styled-components'
+
+import { Row, TemplateBlock } from '@/common/components/storybookParts/previewStyles'
+import { BorderRad, Colors, Transitions } from '@/common/constants'
+
+import {
+  WorkingGroupImagePlaceholder,
+  WGImagePlaceholderProps,
+  WorkingGroupImage,
+} from './WorkingGroupImagePlaceholder'
+
+export default {
+  title: 'WorkingGroup/WorkingGroupImagePlaceholder',
+  component: WorkingGroupImagePlaceholder,
+} as Meta
+
+const Template: Story<WGImagePlaceholderProps> = (args) => {
+  return (
+    <TemplateBlock>
+      <Row>
+        <GroupImageContainer>
+          <WorkingGroupImagePlaceholder {...args} />
+        </GroupImageContainer>
+        <GroupImageContainer>
+          <WorkingGroupImagePlaceholder groupName={undefined} />
+        </GroupImageContainer>
+      </Row>
+      <Row>
+        Group names: bandwidth, storage, forum, membership, content, discovery, operations, gateways or {'{undefined}'}
+      </Row>
+    </TemplateBlock>
+  )
+}
+
+export const WGImagePlaceholder = Template.bind({})
```

Comment:
```suggestion
export const WorkingGroupImage = Template.bind({})
```

Let's keep full names in the code :)